Formula/mongodb-enterprise@6.0.rb

```diff
@@ -0,0 +1,76 @@
+class MongodbEnterprise < Formula
```

Comment:
ditto

Reply:
Maybe this as well. I don't use mongodb enterprise.